ts/Core/Chart/ChartDefaults.ts

```diff
@@ -992,13 +994,61 @@ const ChartDefaults: ChartOptions = {
      * However, `zoomBySingleTouch` will interfere with touch-dragging the
      * chart to read the tooltip. And especially when vertical zooming is
      * enabled, it will make it hard to scroll vertically on the page.
-     * @since 9.0.0
+     * @since next
```

Comment:
Should we change the version here? It was added in 9.0.0, and now it's deprecated- so the version when it was added should stay the same